### DISCUSSION TITLE
Optional / Manual fetch more like in FirestoreQueryBuilder in FirestoreListView

### DISCUSSION THREAD

**Discussion by aadarshadhakalg** [firebase/flutterfire#7739] [Feature request]
Can we add the optional/manual fetch more like in FirestoreQueryBuilder in FirestoreListView also?

For just adding manually fetching more items we have to switch to FirestoreQueryBuilder for now. If we could add a boolean param to the class to fetch more items automatically or not then it would be very easy. 

If this can be done then I can work on this.

Thank You!

**Comment by rrousselGit** [comment]
FirestoreListView is designed to be simple.

Using FirestoreQueryBuilder for more advanced use-cases is the correct solution. It's a tiny bit more code, but you get a lot of freedom with it, so that's worth it.

**Reply by aadarshadhakalg** [reply]
Okay. That's fine. But I think there should be an empty builder like loading builder and itembuilder. The empty builder will be shown when there is no documents.
